src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -1443,6 +1444,10 @@ && ofNullable(caseData.getRespondent2Copy()).isPresent()) {
                     .build());
         }
 
+        if (toggleService.isHmcEnabled()) {
```

Comment:
think this if statement should be removed because this is the only time unavailibility dates gets rolled up.
 if the toggle is off that means this would always be empty even if the toggle is turned on in the future.
if its turned on it would always generate the data.

think it should be removed in all the callback handlers